### ISSUE TITLE
Duplicate tiles get elided from formatted output

### ISSUE THREAD

**Issue by CJLove** [yeastplume/aloevera#15]
I'm still getting up to speed with developing a tileset in Piskel and generating png inputs to aloevera so this might be pilot error.  I initially had some blank/duplicated tiles earlier in the tileset which I hadn't yet defined.  These tiles were elided at some point (either at input time or format time).  Consequently the ordering of subsequent tiles in the generated output (.bin file) was affected.  I think the tileset png that I attached to #14 may demonstrate this.  If all tiles are unique then there is no problem with how aloevera formats the tileset and outputs it.

**Comment by yeastplume** [comment]
Yes, this is on purpose. The CX16 is a very low-resource environment, so redundant tiles are automatically removed from the tileset. Unfortunately this means re-numbering since they're stored in a contiguous space in VERA's VRAM.

Should definitely ensure this behaviour is better documented, thanks!
